homeassistant/components/prometheus/__init__.py

```diff
@@ -90,6 +92,7 @@
 CONF_FILTER = "filter"
 CONF_REQUIRES_AUTH = "requires_auth"
 CONF_PROM_NAMESPACE = "namespace"
+CONF_INCLUDE_EXTRA_LABELS = "include_extra_labels"
```

Comment:
Adding labels is backwards compatible, so I don't think we need a new configuration parameter for this.